econet_base.yaml

```diff
@@ -21,7 +21,7 @@ esphome:
   board: $board
   project:
     name: "esphome-econet.esphome-econet"
-    version: 1.2.1
+    version: 1.3.0
```

Comment:
Wait for https://github.com/esphome-econet/esphome-econet/pull/155 before you make a release